rules/windows/credential_access_mimikatz_powershell_module.toml

```
@@ -4,14 +4,14 @@ maturity = "development"
 updated_date = "2021/05/10"
 
 [rule]
-author = ["Elastic"]
+author = ["Elastic", "Austin Songer"]
 description = """
 Mimikatz is a credential dumper capable of obtaining plaintext Windows account logins and passwords, along with many
 other features that make it useful for testing the security of networks. This rule detects the Invoke-Mimikatz
 PowerShell command.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Mimikatz Powershell Module Activity Detected"
```

while we're on this (development) rule, I'll make a suggestion to remove "Detected", since that's redundant (all rules detect things)
```suggestion
name = "Mimikatz Powershell Module Activity"
```